spirit_utils/src/robot_interface.cpp

```diff
@@ -18,14 +18,12 @@ RobotInterface::RobotInterface(ros::NodeHandle nh) {
 }
 
 void RobotInterface::controlInputCallback(const spirit_msgs::ControlInput::ConstPtr& msg) {
-  // ROS_INFO("In controlInputCallback");
 }
 
 void RobotInterface::publishJointEncoders() {
-  // ROS_INFO("In publishJointEncoders");
   sensor_msgs::JointState msg;
 
-  std::vector<std::string> joint_names = {"0","1","2","3","4","5","6","7","8","9","10","11"};
+  std::vector<std::string> joint_names = {"j8", "j0", "j1", "j9","j2", "j3", "j10", "j4","j5", "j11", "j6", "j7"};
```

Comment:
These don't need to align with the URDF specified joint names do they? Those should still be 0-11.

Reply:
Actually that's worth discussing - I used the urdf with the stl meshes which uses j0 j1 ... for the joint names. We should probably make sure we're consistent about that naming scheme.